oioioi/clock/urls.py

```diff
@@ -5,5 +5,6 @@
 app_name = 'clock'
 
 urlpatterns = [
-    re_path(r'^admin/time/$', views.admin_time, name='admin_time'),
+    # Don't use the 'admin/' prefix, as that sometimes results in breakage.
+    re_path(r'^admin_time/$', views.admin_time, name='admin_time'),
```

Comment:
This should fix admin time not working while in contest right?

Reply:
Yes, though fixing the oioioiadmin namespace warning made admin time fail even outside of contests.